### ISSUE TITLE
Add option to download with format code in addition to best and worst

### ISSUE THREAD

**Issue by kmajeed** [jely2002/youtube-dl-gui#145]
**Is your feature request related to a problem? Please describe.**

When downloading something from YT there are multiple formats  e.g. for a 1080p video I would like to choose a format code of my liking from 137, 248 or 399 rather than relying on the default "best" or "worst option" presented in the app.

```
youtube-dl -F <youtube_video_URL> | grep 1080p

399          mp4        1920x1080  1080p 1091k , mp4_dash container, av01.0.08M.08@1091k, 25fps, video only, 316.76MiB
248          webm       1920x1080  1080p 1341k , webm_dash container, vp9@1341k, 25fps, video only, 389.37MiB
137          mp4        1920x1080  1080p 1585k , mp4_dash container, avc1.640028@1585k, 25fps, video only, 460.13MiB
```

**Describe the solution you'd like**
An drop down list of video/audio formats to choose. so that if I choose 137 (1080p avc) for video and  140 ( m4a) for audio the app downloads these and merges them into single file for me (mp4/mkv etc)   

**Describe alternatives you've considered**
Currently, I use command line version of yt-dl  to do it 

**Additional context**
I am not sure what format the subtitles are added to video. But I would like to be able to save them in specific format e.g as srt format. Currently, I use `--convert-subs srt --embed-subs` command to achieve this from terminal

Similar functionality is provided in [videomass](https://jeanslack.github.io/Videomass/screenshots.html)

**Comment by jely2002** [comment]
The app allows the user to select the resolution and fps. In an upcoming update the user can also select the encoding and audio bitrate. Showing seemingly random numbers to users is confusing and not good for the user experience. This is why I will not be adding this feature to the app.

Tl;dr
The existing format options are sufficient, showing random numbers only confuses the user.
